_posts/2024-12-19-cosmicqc.md

```diff
@@ -0,0 +1,313 @@
+2. The code is available via FigShare, but it was a bit hard to parse. Since it is not on GitHub, that also limits the usability since there isn't an environment or dependencies, and there isn't a way to create issues and contact the maintainers for assistance. As well, I found that the code was not easy to understand and lacked documentation. I want to make clear, I appreciate that they made the code available and open source which is more than some other papers! My main point is that I don't think this method is feasible to include in my standard workflow.
+
+In conclusion, I want to thank the authors of this paper for their hard work and their important contributions to pushing the field towards developing quality control methodology at the single-cell level for morphology profiles.
+
+After the research I conducted and papers I found, I was determined to pursue working on a new methodology, one that is simple, to filter out poor quality segmentations from single-cell morphology profiles.
+
+---
+
+## Introduction to coSMicQC
+
+After 2 and half years of being in my position and performing image-based profiling, I have come to realize that I am not perfect (shocking I know! 😱).
+No matter how much I try to manually optimize my segmentation parameters in CellProfiler or when using [Cellpose](https://www.cellpose.org/) or [Stardist](https://stardist.net/), it will not work for all of the single cells in my datasets.
+Now, I am not necessarily saying this is a fault of my own 😉, but segmentation errors are a persistent issue and it is currently a problem that hasn't been solved to this day.
+
+My point is, you can expect all of your datasets to contain some poorly segmented cells that can be described as technical artifacts.
+You can also expect that debris or smudges could also end up being mis-segmented as cells.
+When working with single-cell data, we do not want these technical artifacts impacting the interpretation of our results.
+Thus, was born this simple idea in my head.
+
+Why not just filter out single cells based on the extracted morphology features?
+
+You wouldn't need to train any model or perform any other pipeline.
+You already have the data in your lap to use, but how you use that data is what is important.
+
+In arriving to the stage, [coSMicQC](https://github.com/WayScience/coSMicQC), or Single cell Morphology Quality Control!
+
+{%
+  include figure.html
+  image="images/blog/cosmicqc/cosmicqc_stage.png"
+  width="100%"
```

Comment:
```suggestion
  width="75%"
```

This figure is currently very large! Please feel free to ignore this, or to play around with this parameter

Reply:
I do love it though!

Reply:
Ohhhh I didn't notice! I agree, let me change it and view the deployment to see if that is better, thanks for pointing out!